json/products/mozilla-concrete-brick.json

```diff
@@ -1,8 +1,11 @@
 {
   "description": {
-    "en": "Such a fine brick"
+    "en": "Brick"
+  }, 
+  "img": "https://raw.githubusercontent.com/mozilla/payments-config/master/payments_config/assets/default.png", 
+  "price": {
+    "en": "$10.00"
   }, 
-  "img": "https://raw.githubusercontent.com/mozilla/payments-config/master/payments_config/assets/brick.png", 
```

Comment:
yeah we can remove the file, I thought it would be useful to have one image and one default, so using the default for brick...